app/views/includes/foot.jade

```diff
@@ -10,20 +10,24 @@ script(type='text/javascript', src='/lib/underscore/underscore-min.js')
 script(type='text/javascript', src='/lib/bootstrap/dist/js/bootstrap.js')
 
 //AngularJS
+
 script(type='text/javascript', src='https://code.angularjs.org/1.1.5/angular.js')
 script(type='text/javascript', src='https://code.angularjs.org/1.1.5/angular-resource.js')
 script(type='text/javascript', src='https://code.angularjs.org/1.1.5/angular-cookies.js')
 
+
```

Comment:
Any reason behind all the extra lines added in this file demarcating each block?